test/unittests/commands/excel2json/test_new_lists.py

```diff
@@ -34,15 +34,15 @@ def test_resolve_duplicates_in_all_excels(self) -> None:
                 "id": ["0", "1", "2"],
                 "en_list": ["List1", "List1", "List1"],
                 "en_1": [pd.NA, "Node1", "Node2"],
-                "ID (optional)": [pd.NA, pd.NA, pd.NA],
+                "id (optional)": [pd.NA, pd.NA, pd.NA],
```

Comment:
As mentioned, I would be okay if you lowercase the column titles for ease of use inside the python code. But I would prefer to have uppercase ID in the Excel

Reply:
The excel remains unchanged, this is because when we clean the excel we make everything lower case so that this is not something that will raise a keyerror. Therefore the dfs that are used after the cleaning must also be lower case.